commit d4940b84da4fbba7f7ed6e6287e1bef48d1293e9
Author: Gaosheng Cui <cuigaosheng1@huawei.com>
Date:   Wed Sep 14 19:04:37 2022 +0800

    x86/kprobes: Remove unused arch_kprobe_override_function() declaration
    
    All uses of arch_kprobe_override_function() have been removed by
    commit 540adea3809f ("error-injection: Separate error-injection
    from kprobe"), so remove the declaration, too.
    
    Link: https://lkml.kernel.org/r/20220914110437.1436353-3-cuigaosheng1@huawei.com
    
    Cc: <mingo@redhat.com>
    Cc: <tglx@linutronix.de>
    Cc: <bp@alien8.de>
    Cc: <dave.hansen@linux.intel.com>
    Cc: <x86@kernel.org>
    Cc: <hpa@zytor.com>
    Cc: <mhiramat@kernel.org>
    Cc: <peterz@infradead.org>
    Cc: <ast@kernel.org>
    Signed-off-by: Gaosheng Cui <cuigaosheng1@huawei.com>
    Signed-off-by: Steven Rostedt (Google) <rostedt@goodmis.org>

diff --git a/arch/x86/include/asm/kprobes.h b/arch/x86/include/asm/kprobes.h
index 71ea2eab43d5..a2e9317aad49 100644
--- a/arch/x86/include/asm/kprobes.h
+++ b/arch/x86/include/asm/kprobes.h
@@ -50,8 +50,6 @@ extern const int kretprobe_blacklist_size;
 
 void arch_remove_kprobe(struct kprobe *p);
 
-extern void arch_kprobe_override_function(struct pt_regs *regs);
-
 /* Architecture specific copy of original instruction*/
 struct arch_specific_insn {
 	/* copy of the original instruction */